### DISCUSSION TITLE
Migrating the structure created using FAST to the latest version

### DISCUSSION THREAD

**Discussion by rmrod** [GoogleCloudPlatform/cloud-foundation-fabric#2442] [General]
Hi guys, first of all, awesome work! We've been using FAST for over a year now.

I understand that FAST does not explicitly support version upgrades. However, is it possible to migrate versions within the same organization?

For instance, is it possible to create a new FAST structure within a parent folder (current version is at the root of the org) with all stages (bootstrap, resman, network, etc.) and migrate my resources gradually?

**Comment by juliocc** [comment]
> For instance, is it possible to create a new FAST structure within a parent folder (current version is at the root of the org) with all stages (bootstrap, resman, network, etc.) and migrate my resources gradually?

You won't be able to apply bootstrap on a folder since it needs to assign some permissions at the org level. All other stages can be applied on different folders if you provide the right input values.

Personally, instead of migrating gradually to a whole new deployment, I'd cherry-pick specific features from the new FAST version and implement them directly into your existing deployment.

**Reply by markjons2** [reply]
A friend of mine did the same, but when i tried picking up all the specific features, it didn't work the same on my [website](https://socialhubin.com/why-i-cant-see-comments-on-threads-problem-solved/), as it worked for him, i had errors displaying.
